decidim-core/app/models/decidim/user.rb

```diff
@@ -6,6 +6,7 @@ module Decidim
   # A User is a citizen that wants to join the platform to participate.
   class User < ApplicationRecord
     MAXIMUM_AVATAR_FILE_SIZE = 5.megabytes
+    ROLES = %w(admin user_manager).freeze
```

Comment:
TLDR; For the users admin is just a role but the implementation is admin is just a flag.

I know it's confusing 😄 . The admin role will not be included in the user roles attribute as we have a flag reserved to it.

We are going to keep iterating in this part because the roles attribute was removed by me in #1605 because it wasn't needed but it come back again for this particular case.

As I discussed with @oriolgual several times I don't consider being an admin a role because it implies all the other available roles. On the other hand a user cannot have more than role right and I am not sure if in the future it will be possible.
We are going to create a new "Users admin" soon (#1597) and I suggest doing some cleaning and refactor there 😄 .